### DISCUSSION TITLE
Migration page.

### DISCUSSION THREAD

**Discussion by lucasrochadejesus** [JoshClose/CsvHelper#2073] [Q&A] [answered]
Hi guys,
Any plan to update migration page? I'm using version 28.0.1 yet and I see we have version 30.
Thanks.

**Comment by JoshClose** [comment, marked as answer]
Oops! The docs were there but I didn't put the links in to get to them. It's updated now.

**Comment by lucasrochadejesus** [comment]
Thanks Josh!!
